phpcs.xml.dist

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0"?>
+<ruleset>
+    <arg name="basepath" value="."/>
+    <arg name="extensions" value="php"/>
+    <arg name="parallel" value="80"/>
+    <arg name="cache" value=".phpcs-cache"/>
+    <arg name="colors" />
+
+    <!-- Ignore warnings, show progress of the run, and show sniff names -->
+    <arg value="nps"/>
+
+    <autoload>.phpcs/autoload.php</autoload>
+
+    <file>.phpcs</file>
+    <file>src</file>
+    <file>tests</file>
+
+    <rule ref="Doctrine">
```

Comment:
This pulls in all rules from the Doctrine coding standard and allows us to exclude/modify some of them. Since we just configure phpcs.xml our ruleset doesn't have a name, that would only happen if we create a ruleset.xml in the standardised directory structure, which isn't necessary if we're not going to distribute the coding standard.